### PR TITLE
refacter_ui_user

### PR DESCRIPTION
**Pull request by hienbx94**
I have created a pull request. Please check it.

### CODE REVIEW

**Comment by trantube94** [comment]
OK

**Comment by lannp** [comment]
OK

**Comment by vietanhbkaaa** [comment]
OK

**Comment by hellovietnam93** [comment]
Ok

File: app/views/courses/show.html.erb [hunk 1]
```diff
@@ -48,8 +48,8 @@
 
       <div class="row">
         <div class="information">
-          <strong><%= t "subjects.start_date" %></strong><br>
-          <strong><%= t "subjects.end_date" %></strong>
+          <strong><%= t "subjects.start_date" %></strong> <%= @course.start_date %><br>
```
Comment (dieunb) [inline app/views/courses/show.html.erb]: Format date

File: app/views/courses/show.html.erb [hunk 2]
```diff
@@ -48,8 +48,8 @@
...
+          <strong><%= t "subjects.end_date" %></strong> <%= @course.start_date %>
```
Comment (dieunb) [inline app/views/courses/show.html.erb]: Format date

**Pull request merged**